instrumentation/bunny/lib/opentelemetry/instrumentation/bunny/patch_helpers.rb

```diff
@@ -12,19 +12,19 @@ module Bunny
       # For additional details around trace messaging semantics
       # See https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md#messaging-attributes
       module PatchHelpers
-        def self.with_send_span(channel, tracer, exchange, routing_key, &block)
+        def self.with_send_span(channel, tracer, exchange, routing_key, &block) # rubocop:disable Style/ArgumentsForwarding
           attributes = basic_attributes(channel, channel.connection, exchange, routing_key)
           destination = destination_name(exchange, routing_key)
 
-          tracer.in_span("#{destination} publish", attributes: attributes, kind: :producer, &block)
+          tracer.in_span("#{destination} publish", attributes: attributes, kind: :producer, &block) # rubocop:disable Style/ArgumentsForwarding
```

Comment:
Since the `tracer.in_span` call has arguments that aren't forwarded, we can't replace `&block` with `...` in the method definition args nor the `tracer.in_span` args. Same is true for example below.